read_command.go

```diff
@@ -146,6 +148,10 @@ func (cmd *readCommand) handleUdfError(resultCode ResultCode) error {
 	return NewAerospikeError(resultCode)
 }
 
+type lazyMapUnpacker struct {
+	*unpacker
+}
```

Comment:
as discussed in the meeting this isn't necessary since the unpacker already supports the `UnpackMap()` interface

Reply:
done